tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -938,38 +1007,21 @@ def _convert(cli_parser: argparse.ArgumentParser, framework, args):
         inp_model_is_object = input_model_is_object(args)
         if inp_model_is_object:
             model_framework = check_model_object(args)
-            if model_framework == "pytorch" and not os.environ.get('USE_PYTORCH_FRONTEND'):
-
-                opset_version = None
-                if 'onnx_opset_version' in args and args['onnx_opset_version'] is not None:
-                    opset_version = args['onnx_opset_version']
-
+            print(args)
+            if model_framework == "pytorch":
+                use_pt_fe = os.environ.get('USE_PYTORCH_FRONTEND') or not args.get("use_legacy_frontend", False)
                 example_inputs = None
                 if 'example_input' in args and args['example_input'] is not None:
                     example_inputs = args['example_input']
+                if not use_pt_fe:
+                    if "use_legacy_frontend" in args:
```

Comment:
some kind of mess of MO args. We need to clarify these cases what we are going to implement